liquidator/liquidator.js

```diff
@@ -49,21 +56,26 @@ class Liquidator {
       Logger.info({
         at: "liquidator",
         message: "liquidating sponsor",
-        address: position.sponsor
+        address: position.sponsor,
+        gasPrice: this.gasEstimator.getCurrentFastPrice()
       });
 
       // Create the liquidation transaction to liquidate the entire position:
-      // - Price to liquidate at (`collateralPerToken`): Since you are determining which positions are under collateralized positions based on the priceFeed, 
+      // - Price to liquidate at (`collateralPerToken`): Since you are determining which positions are under collateralized positions based on the priceFeed,
       // you also should be liquidating using that priceFeed.
       // - Maximum amount of Synthetic tokens to liquidate: Liquidate the entire position.
       liquidationPromises.push(
         this.empContract.methods
           .createLiquidation(
-            position.sponsor, 
-            { rawValue: toWei(priceFeed) },
+            position.sponsor,
+            { rawValue: this.web3.utils.toWei(priceFeed) },
             { rawValue: position.numTokens }
           )
-          .send({ from: this.account, gas: 1500000 })
+          .send({
+            from: this.account,
+            gas: 1500000,
```

Comment:
Is there a workaround to hard-coding `gas`? You could just not specify `gas` (unless you found this was running into the default gas limit set by truffle config) or try calling `.createLiquidation(args).estimateGas({ from: })` which returns a gas cost. https://web3js.readthedocs.io/en/v1.2.0/web3-eth-contract.html#methods-mymethod-estimategas

Reply:
ye, not specifying it is an option. there is no real downside to putting a value here though. I think we need to investigate this more to find a more streamlined awnser. the estimateGas is an option for sure. I'm of the opinion that this can be left for now and updated later if need be. what really matters is the `gasPrice`

Reply:
Agreed. I'm just wondering why we even hardcode `gas` instead of leaving it blank

Reply:
in the initial implementation tx would revert without it. I didnt investigate further after finding that this fixed it 🤷‍♂️

Reply:
interesting, I guess it exceeds the default gas amount